src/components/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
+import { Switch, Route } from 'react-router-dom'
+import Shop from '../../screens/Shop/Shop'
```

Comment:
I was using this article as example https://medium.com/@alexmngn/how-to-better-organize-your-react-applications-2fd3ea1920f1 (i have no idea how scenes become screens 😅  ) for me it sounds more natural because container look like something just to keep other components but here it also have own elements (but maybe it's bad?).